build.sh

```diff
@@ -41,6 +41,6 @@ else
 fi
 
 go build -ldflags "$LDFLAGS" -v ./cmd/openvdc
-go build -ldflags "$LDFLAGS" -v ./cmd/openvdc-scheduler
+go build -ldflags "$LDFLAGS -X 'main.DefaultConfPath=/etc/openvdc/scheduler.conf'" -v ./cmd/openvdc-scheduler
```

Comment:
Why the .conf extension? In other places we're going with `toml` files so I think we should do it here too. I know viper supports multiple extensions and that's fine but let's use .toml for the default file.